### PR TITLE
refact: adopt & use apache version in toolchain

### PR DESCRIPTION
**Pull request by z7658329**
change package to org.apache.hugegraph

### CODE REVIEW

**Comment by javeme** [comment]
ci error:
`Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException: Could not find artifact org.apache.hugegraph:hugegraph-common:jar:1.0.0 in central (https://repo.maven.apache.org/maven2)
`

**Comment by z7658329** [comment]
image

**Comment by z7658329** [comment]
i have reopen some ignored ut, do you know this error before?
image

image

**Comment by imbajin** [comment]
image

and try rerun it

File: hugegraph-tools/pom.xml [hunk 1]
```diff
@@ -38,7 +38,7 @@
         <shell-executable>bash</shell-executable>
         <junit.version>4.12</junit.version>
         <maven.build.timestamp.format>yyyy-MM-dd HH:mm:ssZ</maven.build.timestamp.format>
-        <hugegraph-client-version>2.0.1</hugegraph-client-version>
+        <hugegraph-client-version>1.0.0</hugegraph-client-version>
```
Comment (imbajin) [inline hugegraph-tools/pom.xml:41]: seems we need use ${revision} here? (or enhance it in next PR)

File: hugegraph-client/src/main/java/com/baidu/hugegraph/api/schema/PropertyKeyAPI.java [hunk 2]
```diff
@@ -25,12 +25,12 @@
...
 import com.baidu.hugegraph.client.RestClient;
 import com.baidu.hugegraph.exception.NotSupportException;
-import com.baidu.hugegraph.rest.RestResult;
+import org.apache.hugegraph.rest.RestResult;
 import com.baidu.hugegraph.structure.SchemaElement;
 import com.baidu.hugegraph.structure.constant.HugeType;
 import com.baidu.hugegraph.structure.constant.WriteType;
 import com.baidu.hugegraph.structure.schema.PropertyKey;
-import com.baidu.hugegraph.util.E;
+import org.apache.hugegraph.util.E;
```
[the remark this answers is not in the document]
Comment (z7658329) [reply hugegraph-client/src/main/java/com/baidu/hugegraph/api/schema/PropertyKeyAPI.java:33]: ok

File: hugegraph-tools/pom.xml (hunk 1 above)
Comment (imbajin) [reply hugegraph-tools/pom.xml:41]: oh, no need use a param now (just use inner version param is enough)

**Comment by imbajin** [comment]
why hubble failed?

image

**Comment by z7658329** [comment]
> why hubble failed?
> 
> image

test code compile error

**Review by javeme — APPROVED** [review]

**Pull request merged**